picard/ui/profileeditor.py

```diff
@@ -235,12 +236,74 @@ def make_setting_tree(self, settings=None):
                 child_item.setFlags(QtCore.Qt.ItemIsEnabled | QtCore.Qt.ItemIsUserCheckable)
                 state = QtCore.Qt.Checked if settings and setting.name in settings else QtCore.Qt.Unchecked
                 child_item.setCheckState(self.TREEWIDGETITEM_COLUMN, state)
+                if settings and setting.name in settings and settings[setting.name] is not None:
+                    value = settings[setting.name]
+                else:
+                    value = None
+                child_item.setToolTip(self.TREEWIDGETITEM_COLUMN, self.make_setting_value_text(setting.name, value))
                 widget_item.addChild(child_item)
             self.ui.settings_tree.addTopLevelItem(widget_item)
             if title in self.expanded_sections:
                 widget_item.setExpanded(True)
         self.building_tree = False
 
+    def make_setting_value_text(self, key, value):
+        ITEMS_TEMPLATE = "\n  - %s"
+        NONE_TEXT = _("None")
+        if value is None:
+            return _("Value not set")
+        if key == "selected_file_naming_script_id":
+            return self.get_file_naming_script_name(value)
+        if key == "list_of_scripts":
+            return self.get_list_of_enabled_scripts(key, ITEMS_TEMPLATE, NONE_TEXT)
+        if key == "ca_providers":
+            return self.get_list_of_enabled_ca_providers(key, ITEMS_TEMPLATE, NONE_TEXT)
+        if isinstance(value, str):
+            return '"%s"' % value
+        if type(value) in {bool, int, float}:
+            return str(value)
+        if type(value) in {set, tuple, list, dict}:
+            return _("List of %i items") % len(value)
+        return _("Unknown value format")
+
+    def get_file_naming_script_name(self, script_id):
+        config = get_config()
+        if script_id in config.setting["file_renaming_scripts"]:
+            return config.setting["file_renaming_scripts"][script_id]["title"]
+        presets = {x["id"]: x["title"] for x in get_file_naming_script_presets()}
+        if script_id in presets:
+            return presets[script_id]
+        return _("Unknown script")
+
+    def get_list_of_enabled_scripts(self, key, ITEMS_TEMPLATE, NONE_TEXT):
```

Comment:
Capitalized names are for constants, and constants shouldn't be parameters.

Instead define them at class level.
Also since it's specific to _list_of_scripts_ we don't even need to pass `key`.

```python
def get_list_of_enabled_scripts(self):
    scripts = config.setting['list_of_scripts']
    ....
           value_text += self.ITEMS_TEMPLATE % name
 ```